File: CTDK_Python/106/_106.py
from math import pi


x = float(input("Nhap x: "))

xx = (x * pi) / 180
s = xx
t = xx
m = 1
dau = -1
i = 3
e = xx
epsilon = 10**-6

while (e >= epsilon):
    t = t * xx * xx
    m = m * i * (i - 1)
    e = t / m
    s = s + dau * e
    dau = -dau
    i = i + 2
    
print("sin(x) = ", s)

File: CTDK/106/106.cpp
#include <iostream>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	float xx = (x * 3.14) / 180;
	float s = 1;
	float t = 1;
	int m = 1;
	int dau = -1;
	float e = 1;
	int i = 2;
	while (e >= pow(10, -6))
	{
		t *= xx * xx;
		m *= i * (i - 1);
		e = t / m;
		s += dau * e;
		dau = -dau;
		i += 2;
	}
	cout << "S = " << s << endl;
	return 0;
}